cost-analyzer/templates/cost-analyzer-cluster-role-binding-template.yaml

```diff
@@ -33,4 +33,21 @@ subjects:
   - kind: ServiceAccount
     name: {{ template "cost-analyzer.serviceAccountName" . }}
     namespace: {{ .Release.Namespace }} 
+---
 {{- end }} 
+{{- if and .Values.global.platforms.openshift.enabled .Values.global.platforms.openshift.createMonitoringClusterRoleBinding }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "cost-analyzer.serviceAccountName" . }}-openshift-monitoring
+  labels:
+    {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view
```

Comment:
What ClusterRole is this referring to? I couldn't find any reference to this in the Helm chart. Is this an existing ClusterRole that's specific to OpenShift installs?

Reply:
yes its specific to openshift installs.
https://docs.redhat.com/en/documentation/openshift_container_platform/4.2/html/monitoring/cluster-monitoring#monitoring-accessing-prometheus-alerting-ui-grafana-using-the-web-console_accessing-prometheus

Reply:
Awesome, thanks for clarifying! Lets leave a comment in the template?

Reply:
sure :+1: